CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - `sequence`
 - Added support for registering and calling Stored Procedures with `TABLE` return type.
```

Comment:
"Stored Procedures" -> "stored procedures"

reason: you only need to capitalize proper nouns and this is not technically a product name.